### PR TITLE
feat: add display options to navigation

### PR DESCRIPTION
**Pull request by illuminati1911**
Adds display options to navigation:
```typescript
/**
 * Defines the options used by the Navigator for dislaying different elements.
 */
export interface DisplayOptions {
  /**
   * Configures whether destination markers are shown during navigation.
   * If true, destination markers are shown.
   */
  showDestinationMarkers?: boolean;
  /**
   * Configures whether stop signs are shown during navigation.
   * If true, stop signs are shown.
   */
  showStopSigns?: boolean;
  /**
   * Configures whether traffic lights are shown during navigation.
   * If true, traffic lights are shown.
   */
  showTrafficLights?: boolean;
}
```

- [ ] Tests pass
- [x] Appropriate changes to documentation are included in the PR

### CODE REVIEW

**Review by jokerttu — CHANGES_REQUESTED** [review]
Small nits

File: android/src/main/java/com/google/android/react/navsdk/NavModule.java [hunk 1]
```diff
@@ -443,10 +457,18 @@ public void setDestinations(ReadableArray waypoints, @Nullable ReadableMap routi
     }
 
     if (routingOptions != null) {
-      pendingRoute =
-          mNavigator.setDestinations(
-              mWaypoints,
-              ObjectTranslationUtil.getRoutingOptionsFromMap(routingOptions.toHashMap()));
+      if (displayOptions != null) {
```
[the remark this answers is not in the document]
Comment (illuminati1911) [reply android/src/main/java/com/google/android/react/navsdk/NavModule.java:460]: > seems like displayOptions are only used when routingOptions are given. so it is not possible to give only displayOptions?

On Android there is no API call for that. So display options are only available if routing options are provided.

File: android/src/main/java/com/google/android/react/navsdk/NavModule.java [hunk 2]
```diff
@@ -403,16 +403,27 @@ private void createWaypoint(Map map) {
   }
 
   @ReactMethod
-  public void setDestination(ReadableMap waypoint, @Nullable ReadableMap routingOptions) {
+  public void setDestination(
```
[the remark this answers is not in the document]
Comment (illuminati1911) [reply android/src/main/java/com/google/android/react/navsdk/NavModule.java:406]: Done.

File: android/src/main/java/com/google/android/react/navsdk/NavModule.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (jokerttu) [reply android/src/main/java/com/google/android/react/navsdk/NavModule.java:435]: No, I think this is fine as this is limitation from the sdk below.
Anyway, I think developers always give routeoptions as those are quite important.

**Review by jokerttu — APPROVED** [review]
LGTM

**Pull request merged**